export/export_pipeline/compute_gltf_representation.py

```diff
@@ -0,0 +1,270 @@
+        constraint_dicts.append(pc._asdict())
+      if n_prismatic_dofs > 1:
+        raise Warning(f'{actor_name} has a multi-axis prismatic constraint')
+      
+      # revolute joints
+      n_revolute_dofs = 0
+      cone_limit = constraint_profile.get_editor_property('cone_limit')
+      pc_args = [parent, child, ]
+      if (cone_limit.get_editor_property('swing1_motion') != unreal.AngularConstraintMotion.ACM_LOCKED):
+        n_revolute_dofs += 1
+        if n_prismatic_dofs > 1:
+          raise Warning(f'{actor_name} already has {n_prismatic_dofs} prismatic joints')
+        limit = cone_limit.get_editor_property('swing1_limit_degrees') * ROT_UE_TO_MUJOCO_SCALE
+        offset = angular_offset.yaw * ROT_UE_TO_MUJOCO_SCALE
+        axis = unreal.Vector.UP
+        axis = axis.rotate(rot).normal()
+        pc_args.extend([
+          f'{component.get_name()}_z_revolute',
+          [-limit, limit],
+          offset,
+          [axis.x, axis.y, axis.z],
+          'revolute'
+        ])
+        pc = PhysicsConstraint(*pc_args)
+        constraint_dicts.append(pc._asdict())
+      pc_args = [parent, child, ]
+      if (cone_limit.get_editor_property('swing2_motion') != unreal.AngularConstraintMotion.ACM_LOCKED):
+        n_revolute_dofs += 1
+        if n_prismatic_dofs > 1:
+          raise ValueError(f'{actor_name} already has {n_prismatic_dofs} prismatic joints')
+        limit = cone_limit.get_editor_property('swing2_limit_degrees') * ROT_UE_TO_MUJOCO_SCALE
+        offset = angular_offset.pitch * ROT_UE_TO_MUJOCO_SCALE
+        axis = unreal.Vector.RIGHT
+        axis = axis.rotate(rot).normal()
+        pc_args.extend([
+          f'{component.get_name()}_y_revolute',
+          [-limit, limit],
+          offset,
+          [axis.x, axis.y, axis.z],
+          'revolute'
+        ])
+        pc = PhysicsConstraint(*pc_args)
+        constraint_dicts.append(pc._asdict())
+      twist_limit = constraint_profile.get_editor_property('twist_limit')
+      pc_args = [parent, child, ]
+      if twist_limit.get_editor_property('twist_motion') != unreal.AngularConstraintMotion.ACM_LOCKED:
+        n_revolute_dofs += 1
+        if n_prismatic_dofs > 1:
+          raise ValueError(f'{actor_name} already has {n_prismatic_dofs} prismatic joints')
+        limit = twist_limit.get_editor_property('twist_limit_degrees') * ROT_UE_TO_MUJOCO_SCALE
+        offset = angular_offset.roll * ROT_UE_TO_MUJOCO_SCALE
+        # UE applied X rotations are right handed
+        # https://github.com/ethz-asl/unreal_airsim/blob/master/docs/coordinate_systems.md
+        axis = unreal.Vector.BACKWARD
+        axis = axis.rotate(rot).normal()
+        pc_args.extend([
+          f'{component.get_name()}_x_revolute',
+          [-limit, limit],
+          offset,
+          [axis.x, axis.y, axis.z],
+          'revolute'
+        ])
+        pc = PhysicsConstraint(*pc_args)
+        constraint_dicts.append(pc._asdict())
+    
+    if len(constraint_dicts) > 0:
```

Comment:
If these json files are flat lists, I'd prefer reading and writing CSV files using pandas for simplicity and consistency.